lib/etso/adapter.ex

```diff
@@ -30,7 +32,7 @@ defmodule Etso.Adapter do
     {:ok, repo} = Keyword.fetch(config, :repo)
     child_spec = __MODULE__.Supervisor.child_spec(repo)
     adapter_meta = %__MODULE__.Meta{repo: repo}
-    {:ok, child_spec, adapter_meta}
+    {:ok, child_spec, Map.from_struct(adapter_meta)}
```

Comment:
Just wondering, if this is necessary (Ecto forced adapter_meta to be a map for a version, then I think they reverted it, and we are pinning a higher version — See 1st point [here](https://github.com/elixir-ecto/ecto/blob/master/CHANGELOG.md#bug-fix-2))

Reply:
Hmmm. Thank you for pointing this out.